utils/dockerutil.py

```diff
@@ -588,7 +588,7 @@ def image_name_extractor(self, co, short=False):
         return image
 
     def image_name_resolver(self, image):
-        if image.startswith('sha256:') or '@sha256:' in image:
```

Comment:
I'm afraid we're breaking the kubernetes `redis@sha256:474fb41b08bcebc933c6337a7db1dc7131380ee29b7a1b64a7ab71dad03ad718` case here, WDTY?

Reply:
Indeed, I wasn't aware of that particular case. I updated the code appropriately to fetch the tag in the _sha256_ when absent before the `@sha256`+